### PR TITLE
[DOCFIX] Add transformation definition docs

### PR DESCRIPTION
**Pull request by bradyoo**

### CODE REVIEW

**Comment by bradyoo** [comment]
@gpang

File: docs/en/operation/User-CLI.md [hunk 1]
```diff
@@ -1363,7 +1363,15 @@ format.
...
 > For the coalesce feature, by default it will coalesce into a maximum of 100 files, 
-> with each file no smaller than 2GB. 
+> with each file no smaller than 2GB.
+ 
+The definition format takes a form of configuration separated by semicolon and specifies the details of the output
+format. Available configurations are:
+
+```console
+file.count.max=<num> (maximum number of files in transformed output)
+file.size.min=<num> (minimum file size of the output)
```
Comment (gpang) [inline docs/en/operation/User-CLI.md:1373]: Is this is bytes? If so, please specify this doc.

**Review by gpang — CHANGES_REQUESTED** [review]
@bradyoo I just had a minor comment.

**Comment by bradyoo** [comment]
Good catch. Fixed.

**Review by gpang — APPROVED** [review]
LGTM

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Comment by gpang** [comment]
alluxio-bot, cherry-pick this to branch-2.3 please

**Pull request merged**